src/providers/toast.service.ts

```diff
@@ -14,4 +18,24 @@ export class ToastService {
     this.faveToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
     this.faveToast.present();
   }
+  locationToast():void{
+    let txt = 'Unable to retrieve current location';
```

Comment:
Hey @artarazavi, so it looks like all of these toast calls are the same except for the message `txt.` I'm thinking we could abstract this out. More specifically, have one toast method and pass a specific `txt` to that method. What do you think? Just offering up a discussion.

Reply:
Open to @akaplo, too

Reply:
Yes! Great idea @mboneil10.